content/concepts/interoperability/cross-chain-bridges.md

```diff
@@ -0,0 +1,127 @@
+
+1. Claim a cross-chain claim ID on the issuing chain.
+2. Submit a cross-chain transfer transaction on the locking chain, attaching the claimed cross-chain claim ID and include a reward for the witness servers. This locks or burns the asset on the issuing chain.
+3. Obtain the attestations from the witness servers that the transaction occurred on the issuing chain.
+4. Submit a transaction claim for the transferred value on the issuing chain, attaching the attestation as proof that the value was indeed transferred.
+5. The rewards are then distributed to the witness servers' accounts on the issuing chain.
+
+Consider an example where Alice wants to send XRP from her account on the XRP Ledger mainchain (locking chain) to her account sAlice on a sidechain (issuing chain).
+
+* A bridge that transfers XRP between the XRP Ledger mainchain and the sidechain has been set up with XChainCreateBridge.
+* Witness servers are up and running.
+* Alice has also already run a XChainCreateAccountCommit transaction at some point, so she has an account on the sidechain.
+
+<!-- Add image of just the bridge created-->
+
+![Cross-chain Transactions](img/xrpl-bridging-solution.png "Cross-chain transactions")
+
+1. sAlice first checks out a claim ID with XChainCreateClaimID on the sidechain, specifying the above bridge. She retrieves the claim ID from the transaction metadata or the TBD RPC call.
+
+2. Alice then takes the cross-chain claim ID from sAlice’s XChainCreateClaimID transaction and submits a XChainCommit transaction on the mainchain with that claim ID, locking up a specified amount of XRP. She specifies sAlice’s account in the OtherChainDestination field.
+
+3. The witnesses then take note of the XChainCommit transaction and submit XChainAddAttestation transactions on the sidechain, attesting to the fact that the XChainCommit transaction did in fact occur on the mainchain.
+
+4. When there are enough XChainAddAttestation signatures to reach quorum, the XRP is automatically released on the sidechain to sAlice’s account.
+
+5. If the XRP is not automatically released, for whatever reason (such as Alice forgetting to specify sAlice’s account in the OtherChainDestination field), then sAlice submits a XChainClaim transaction on the sidechain, specifying her account as the destination. This then releases the XRP on the sidechain to sAlice’s account.
+
+
+## Terminology
+
+* Bridge: A method of moving assets/value from one blockchain to another.
+
+* Witness: Independent servers that are aware of the locking and issuing chains. See [witness servers](witness-server.md) for more information.
+
+* Cross-chain transfer: A transfer of assets from one chain to another.
+
+* Source chain: The chain that a cross-chain transfer begins from. The transfer is from the source chain and to the destination chain.
+
+* Destination chain: The chain that a cross-chain transfer ends at. The transfer is from the source chain and to the destination chain.
```

Comment:
The phrase "ends at" makes me wonder if you can have cross-chain transfers that involve more than two chains. Can you do sidechain → mainchain → other sidechain for example? (I mean all in one and not just with multiple manual steps between)